The-Chonky-Panda/Atelier
# Review modal characteristics

## Description

Added characteristic rating to the write new review modal. 

## How to test

Click on `Add a review` button, when the modal pops up, there should be a section asking the user to rate each characteristic.

On select a value, the current selection would change to the corresponding value.
Uncheck the same radio button would clear current selection.

<img width="680" alt="Screen Shot 2021-09-22 at 5 50 37 PM" src="https://user-images.githubusercontent.com/43324065/134432867-8aae2d85-d94a-4bcd-988d-92365b23862b.png">


## Notes


## Issue tracking

https://trello.com/c/Xs4L3U29
